### ISSUE TITLE
Profile Button Causes Lag

### ISSUE THREAD

**Issue by WenXin20** [McJty/LostCities#245]
`Minecraft v1.14.4`
`Forge v28.1.115`
`Lost Cities v2.0.28`

In my modpack of about 140 mods, I get a huge lag spike every time I press the button to try to change the profile city type.
I've also noticed this lag with fewer mods, maybe 20 mods, although the lag is not as noticable.

If the city preview the cause of this lag, can an option to turn off the preview be added?

**Comment by ghost** [comment]
Seconded - the city preview is causing an extreme amount of lag for me.

**Comment by McJty** [comment]
Yes that's know. It's actually generating terrain to create that preview. Not much that can be done about that. It's only during world creation so I don't think it's a big deal to be honest

**Comment by ghost** [comment]
Would be nice if we could just toggle it off though. I feel I'd rather not cause almost completely crippling lag.

Also, it's definitely possible to do it without lag because TerraForged can preview a rather large area without any major performance hit. Not sure of the process or difficulty, but it certainly isn't impossible.

**Comment by McJty** [comment]
Well city worldgen is considerably more complicated then normal worldgen. You will notice that the profile preview goes a lot faster if you have almost no cities

**Comment by ghost** [comment]
Alright, that's fair enough. I shouldn't assume something can just work better lol, you have a lot more experience than I do. A visual toggle would still solve the issue pretty well!
